Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3
+FROM python:3.8.2
```

Comment:
was this tested? Is there a way this could be breaking?

Reply:
I believe that the psycopg2 in `requirements.txt` doesn't support the latest version of python. Will close this since the latest commits to master already include the changes.